include/behaviortree_cpp/basic_types.h

```diff
@@ -331,21 +331,29 @@ class PortInfo: public TypeInfo
   template <typename T>
   void setDefaultValue(const T& default_value) {
     default_value_ = Any(default_value);
+    has_dyn_default_value_ = false;
```

Comment:
Unless there is a clear motivation, I would rather remove this

Reply:
The motivation is that someone could maybe set the default value after the input port creation with `Any`.